project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module

```diff
@@ -46,7 +46,8 @@ function c4m_features_homepage_c4m_suggested_groups_get_groups() {
   $query
     ->entityCondition('entity_type', 'node')
     ->entityCondition('bundle', 'group')
-    ->propertyCondition('status', NODE_PUBLISHED);
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->fieldCondition('c4m_og_status', 'value', 'published');
```

Comment:
This is probably not true, since a GO should see his `pending` groups as well for example.